core/templates/pages/exploration-editor-page/editor-tab/state-param-changes-editor/state-param-changes-editor.component.ts

```diff
@@ -25,19 +25,12 @@ require(
   'components/state-editor/state-editor-properties-services/' +
   'state-property.service.ts');
 
-angular.module('oppia').directive('stateParamChangesEditor', [
-  'UrlInterpolationService', function(UrlInterpolationService) {
-    return {
-      restrict: 'E',
-      scope: {},
-      templateUrl: UrlInterpolationService.getDirectiveTemplateUrl(
-        '/pages/exploration-editor-page/editor-tab/' +
-        'state-param-changes-editor/state-param-changes-editor.directive.html'),
-      controller: [
-        '$scope', 'StateParamChangesService',
-        function($scope, StateParamChangesService) {
-          $scope.StateParamChangesService = StateParamChangesService;
-        }
-      ]
-    };
-  }]);
+angular.module('oppia').component('stateParamChangesEditor', {
```

Comment:
Remove require(....url-inter...) if not required! Also, change directive to component in file overview!

Reply:
Sorry I didn't understand the first part.

Reply:
Done.